flocker/node/agents/blockdevice.py

```diff
@@ -1490,6 +1569,12 @@ def discover_state(self, node_state):
             NodeState(
                 uuid=self.node_uuid,
                 hostname=self.hostname,
+                # FLOC-1874 The sizes of the datasets for these manifestations
```

Comment:
I removed the comment and added another noting that we don't report the size of non-manifest datasets.